jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyRunWarMojo.java

```diff
@@ -263,6 +263,7 @@ public void restartWebApp(boolean reconfigure) throws Exception
                 break;
             }
             case HOME:
+            case DISTRO:
```

Comment:
Please produce a warning log message if using DISTRO, encouraging use of HOME instead.